### DISCUSSION TITLE
Change the default filter of highfive

### DISCUSSION THREAD

**Discussion by LauZanMo** [BlueBrain/HighFive#977] [General]
I would like to ask whether there is a way to change the default filter of highfive to blosc, because the default filter,which is set to szip, is slow and my system has real-time requirements. I notice that the highfive code does not seem to support the setting of filter. Can it be implemented using the native HDF5 function ?

**Comment by 1uc** [comment]
You've explained it very nicely here: https://github.com/BlueBrain/HighFive/issues/978

**Comment by 1uc** [comment]
I feel closed discussion are harder to find than open ones. Therefore, since this question is valid and potentially useful to other users and even has a good answer, I'd like it to stay open so it's part of the FAQ (aka Discussions).
